### PR TITLE
New javascript question added (typeof Null)

### PR DESCRIPTION
**Pull request by Tathagat27**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

#14

### CODE REVIEW

**Comment by Tathagat27** [comment]
hey @jdwilkin4 , Please have a look.

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1018,6 +1018,17 @@ const javascriptQuiz = [
     Explanation: "DOM stands for Document Object Model.",
     Link: "https://www.freecodecamp.org/news/what-is-the-dom-document-object-model-meaning-in-javascript/",
   },
+  {
+    Question:
+      "When an operator value is NULL, the typeof returned by the operator is:",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1023]: Please start this sentence with In JavaScript.

```suggestion
      "In JavaScript, When an operator value is NULL, the typeof returned by the operator is:",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Tathagat27** [comment]
**Thanks** @jdwilkin4 for your suggestion. I've implemented those changes. Please have a look :)

**Review by jdwilkin4 — APPROVED** [review]
Hi @Tathagat27 ! 

Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**